src/main/java/frc/team2412/robot/commands/index/IndexCommand.java

```diff
@@ -13,13 +13,19 @@ public IndexCommand(IndexSubsystem subsystem) {
 
     @Override
     public void execute() {
-        subsystem.ingestMotorIn();
-        subsystem.feederMotorIn();
+        if (!subsystem.feederSensorHasBallIn()) {
+            subsystem.ingestMotorIn();
+            subsystem.feederMotorIn();
+        } else {
+            subsystem.feederMotorStop();
+            subsystem.ingestMotorStop();
+        }
+
     }
 
     @Override
-    public void end(boolean cancel) {
-        subsystem.ingestMotorStop();
-        subsystem.feederMotorStop();
```

Comment:
Do we want the motors to stop when the button's released?

Reply:
This command is the new default command for index and will no longer be bound to a button